src/commands/migrate.js

```diff
@@ -8,26 +8,43 @@ const migrate = (context, fromVersion, toVersion, optionalPercent = '100%') => {
   optionalPercent = parseInt(optionalPercent, 10);
   return utils.getLinkedApp()
     .then(app => {
-      context.line(`Getting ready to migrate your app "${app.title}" from ${fromVersion} to ${toVersion}.\n`);
-      utils.printStarting(`Starting migration from ${fromVersion} to ${toVersion} for ${optionalPercent}%`);
+      const body = {
+        percent: optionalPercent,
+      };
+      const user = global.argOpts.user;
+
+      if (user) {
+        body.percent = 100;
+        body.user = user;
```

Comment:
Command-line parameters `percent` and `user` shouldn't co-exist. So I think we should block users if they specify both the parameters instead of setting `percent` to 100 silently. What do you think?

Reply:
I can see how that makes sense (maybe they did want the percentage instead). I'll make that change!

Reply:
Well, apparently we can't tell if the percentage was set or is the default one (since we default it to `100%`), so I don't think I can do what you're saying without changing the code around a bit, and it doesn't seem worth it.

What do you think?

Reply:
How about we block users only if `percent != 100` and `user` is specified?